eox_hooks/actions_handler.py

```diff
@@ -0,0 +1,46 @@
+"""This module defines the handlers for the actions triggered by a signal received by eox-hooks."""
+import logging
+from importlib import import_module
+
```

Comment:
```suggestion
from eox_hooks.tests.test_utils import  custom_action_mock
```
This will work with future suggestions, also you can delete DEFAULT_ACTIONS_MODULE and DEFAULT_ACTION_NAME